src/witan/cic/episodes.clj

```diff
@@ -909,13 +909,59 @@
 (defn mark-periods-that-start-before-min-report-year-xf [min-report-year]
   (map (partial mark-periods-that-start-before-min-report-year min-report-year)))
 
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Fix episodes that take place after the extract date
+
+(defn episode-starts-after-extract-date? [extract-date date]
+  (if (nil? date)
+    false
+    (t/>
+     date
+     extract-date)))
```

Comment:
I'd probably nil pun the date and usually prefer to write comparisons as increasing rather than decreasing, but this is OK.

I'd probably do it as

```
(when date
  (t/< extract-date date))
```

Reply:
remind me what nil pun means again?

Reply:
I don't really understand your comment about increasing and decreasing

Reply:
nil punning here is using nil as a `false`-y so you can just check it directly in `if` or `when` and return a a non-nil value or `nil` if you want to return true or false.

Increasing is `<` and decreasing is `>` as you can give them more than just 2 values, so less-than and greater-than doesn't really map. `(< 1 2 3)` is true because 1, 2 and 3 are increasing in value.

Reply:
okay I didn't know about increasing and decreasing. That seems silly, as they are widely recognised symbols, but your argument is definitely valid 👍